rllib/algorithms/dreamerv3/tf/dreamerv3_tf_learner.py

```diff
@@ -53,40 +53,39 @@ def configure_optimizers_for_module(
         dreamerv3_module = self._module[module_id]
 
         # World Model optimizer.
-        optim_world_model = tf.keras.optimizers.Adam(
-            learning_rate=hps.world_model_lr, epsilon=1e-8
-        )
+        optim_world_model = tf.keras.optimizers.Adam(epsilon=1e-8)
         optim_world_model.build(dreamerv3_module.world_model.trainable_variables)
         params_world_model = self.get_parameters(dreamerv3_module.world_model)
         self.register_optimizer(
             module_id=module_id,
             optimizer_name="world_model",
             optimizer=optim_world_model,
             params=params_world_model,
+            lr_or_lr_schedule=hps.world_model_lr,
         )
 
         # Actor optimizer.
-        optim_actor = tf.keras.optimizers.Adam(learning_rate=hps.actor_lr, epsilon=1e-5)
+        optim_actor = tf.keras.optimizers.Adam(epsilon=1e-5)
```

Comment:
Why are we hardcoding the lr as opposed to reading it from config with this default value?

Reply:
We are setting the lr anyways to the configured value (or schedule) a few lines down below. So there is no need to set it via the constructor.

Reply:
Oh sorry I thought we were setting lr to 1e-5. I miss read the code. 👍